resume.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: page
+title: ''
+---
+
+If you want a pdf version of the resume to take with you, **[click here]({{ site.url }}/assets/resumes/resume.pdf){: .btn}**
+{: .message }
+
+<!-- JEKYLL-REMOVE-END -->
```

Comment:
Experimenting with removing Jekyll specific markdown content so it doesn't show up in the generated pdf resume